RayLin.md

```diff
@@ -1,29 +1,26 @@
-# 08/01 ~ 08/05
+# 08/08 ~ 08/12
 
 - [Video Control]
   - Bug 1291013 - Closed-caption button spacing is asymetric
-    - review cancelled, :jaws is overloaded
-    - ask again after flag removed
+    - FIXED
   - Bug 1291009 - Closed-caption button fuzzy in hidpi mode
-    - r+, :jaws
-  - Bug 1291268 - HiDPI image are missing from video controls on Windows and Linux
-    - code review, granted and landed
-  - Bug 1292083 - Split mobile/desktop video control and make them inherited from a minimal base binding
-    - WIP
-    - :jaws agrees with this idea.
+    - FIXED
+  - Bug 1293601 - \[Fennec\] The video controls are missing the time slider and show a random number at the start of video playback on YouTube
+    - r+, checkin-needed
   - Bug 1271765 - Visual refresh of media controls
-    - Stephen reviewed the concept and mainly agreed
-    - Peko will offer icons in SVG format first and then spec
-  - Bug 1222273 - Convert HTML \<video\>/\<audio\> controls to be (mostly?) HTML instead of XUL
-    - Depends on: 1292083
+    - WIP
+    - :dolske and :jaws gave some feedback.
+    - Got a spec draft from Peko.
+  - Bug 1292083 - Split mobile/desktop video control and make them inherited from a minimal base binding
+    - Deferred
```

Comment:
I was planning to start visual refresh once I got spec and icons. De-XUL process is flexible and doesn't conflict with visual refresh, so I could easily switch back and continue prior work.